SEED-platform/seed
# BuildingSync Asset Extractor Functionality

#### What's this PR do?
This PR implements the BAE functionality in seed automatically on buildingsync file imports

#### How should this be manually tested?
- Run database migrations
- Import a BuildingSync file and notice the assets extracted and mapped

#### What are the relevant tickets?
Resolves #3200
#### Screenshots (if appropriate)
<img width="1506" alt="Screen Shot 2022-04-18 at 12 23 13 PM" src="https://user-images.githubusercontent.com/2205659/164359450-493c39f5-42e0-4fa8-9d2b-50d023cfb8e5.png">
